src/datepicker/datepicker.js

```diff
@@ -253,7 +262,6 @@ export default class Datepicker<T = Date> extends React.Component<
     }
 
     if (range && !separateRangeInputs) {
-      // TODO(LUKE): use INPUT_DELIMITER below
```

Comment:
use `INPUT_DELIMITER` like the code above?

Reply:
So it's unclear to me whether `INPUT_DELIMITER` should be ` – ` (as it is now) or `–`, which also shows up. For now I used the former, and only in cases that pertained to the dates themselves (rather than to the mask). To completely remove hardcoded en dashes from the file, I should use the latter instead, which would mean using spaces around INPUT_DELIMITER in many cases, as in ` ${INPUT_DELIMITER} `. I'll save this change for a separate PR.

Reply:
Changed my mind, I'll switch to using `INPUT_DELIMITER = '–'` for this PR.